docs/cvl/v5-changes.md

```diff
@@ -193,6 +194,52 @@ is the main contract being verified, then passing `--contract C` will cause
 method variables to be instantiated in the same way the would have in older
 versions.
 
+Disallow Solidity function calls in quantified expressions
+----------------------------------------------------------
+
+Starting with `certora-cli` version 5.0, the Prover no longer supports
+making Solidity contract `method` calls in quantified expression bodies by
```

Comment:
not sure we need to backtick `method` here

Reply:
Changed.